apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/CompiledGraphQL.kt

```diff
@@ -307,7 +314,19 @@ class CompiledVariable(val name: String)
  *
  * Note: for now, enums are mapped to Strings
  */
-class CompiledArgument(val name: String, val value: Any?, val isKey: Boolean = false)
+class CompiledArgument @ApolloExperimental constructor(
+    val name: String,
+    val value: Any?,
+    val isKey: Boolean = false,
+    @ApolloExperimental
+    val isPagination: Boolean = false,
```

Comment:
Two parameters with default values => Sounds like it's time for a `Builder`?

Reply:
Let's go :)